game/osu/DateTime.ksy

```diff
@@ -0,0 +1,9 @@
+meta:
+  id: datetime
+  endian: le
+seq:
+  - id: ticks
+    type: u8
+instances:
+  value:
+    value: ticks / 1000000

```

Comment:
the result will be an integer. Is it intended?

Reply:
Division by `1000000.0` is probably easier.